Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = 'Apache-2.0'
 name = "hydra-dx-math"
 description = "A collection of utilities to make performing liquidity pool calculations more convenient."
 repository = 'https://github.com/galacticcouncil/hydradx-math'
-version = "4.2.0"
+version = "4.2.1"
```

Comment:
This is a breaking change

Reply:
Do we want to create a new PR for this?

Reply:
and it was doing anything useful anymore.

Reply:
> it is not breaking change. the fee struct was only used internally in the math crate.

Oh. Then sorry I was sure I saw this on the frontend somewhere but I might have been wrong.